check_isa.cpp

```diff
@@ -32,20 +38,33 @@ static void __cpuidex(int feature_regs[4], int level, int count) {
                           : "=a" (feature_regs[0]), "=r" (feature_regs[1]), "=c" (feature_regs[2]), "=d" (feature_regs[3])
                           : "0" (level), "2" (count));
 }
+#endif // !WINDOWS
 
 static int call_xgetbv() {
+#ifndef WINDOWS
     // Call xgetbv. We use byte sequence in order to support older assemblers.
     int eax, edx;
-    __asm__ __volatile__ (".byte 0x0f, 0x01, 0xd0" : "=a" (eax), "=d" (edx) : "c" (0));
```

Comment:
What about just borrowing check_isa.cpp from ISPC project?
https://github.com/ispc/ispc/blob/master/check_isa.cpp

It was originally taken from there.

Reply:
Fixed